zos_concepts/gdg_datasets/copy_edit_fetch/README.md

```diff
@@ -0,0 +1,89 @@
+# Create add edit copy and fetch with Generation data group
+This playbook demonstrates how to create a GDG edit copy and fetch
+creations of GDS.
```

Comment:
Would rephrase this to "[...] to create a Generation Data Group (GDG), edit, copy and fetch different Generation Data Sets (GDSs) in the GDG.